erpnext/accounts/doctype/bank_transaction/bank_transaction.py

```diff
@@ -8,8 +8,23 @@
 
 
 class BankTransaction(StatusUpdater):
+	_cy_precision = None
+
+	def flt(self, value):
+		"frappe needs a money datatype"
+		if self._cy_precision is None:
+			import math
+
+			fraction_units = (
+				frappe.db.get_value("Currency", self.currency, "fraction_units", cache=True) or 100
+			)
+			self._cy_precision = math.log(fraction_units, 10)
```

Comment:
`log(1000, 10) != 3`
```suggestion
			self._cy_precision = round(math.log(fraction_units, 10))
```

`math.log` returns a float, while `flt` expects an integer as precision.